crates/orchestrator/src/cli/alert/aws_sns.rs

```diff
@@ -0,0 +1,14 @@
+use clap::Args;
+
+/// Parameters used to config AWS SNS.
+#[derive(Debug, Clone, Args)]
+#[group()]
```

Comment:
As discussed, there's a default value available here

```rust
pub struct AWSSNSCliArgs {
    /// Use the AWS SNS client
    #[arg(long)]
    pub aws_sns: bool,

    /// The name of the S3 bucket.
    #[arg(env = "MADARA_ORCHESTRATOR_AWS_SNS_ARN", long, default_value = Some("madara-orchestrator-arn"))]
    pub sns_arn: Option<String>,
}
```

Reply:
require_all means we'll have to provide the value from cli, 
looking at the code anyone would assume that if not provided it will pick from env or default but it doesn't.